tensorflow_graphics/rendering/camera/perspective.py

```diff
@@ -201,40 +224,43 @@ def intrinsics_from_matrix(matrix, name=None):
       "perspective_intrinsics_from_matrix".
 
   Returns:
-    Tuple of two tensors, each one of shape `[A1, ..., An, 2]`. The first
-    tensor represents the focal length, and the second one the principle point.
+    Tuple of three tensors, first two of shape `[A1, ..., An, 2]` and the third of shape `[A1, ..., An, 1]`. The first
+    tensor represents the focal length, and the second one the principle point. The third represent Skew Coefficient.
```

Comment:
nit on docstring: how about "The first tensor represents the focal length, the second the principal point, and the third the skew."